src/main/java/com/arpnetworking/metrics/mad/configuration/AggregatorConfiguration.java

```diff
@@ -331,6 +352,8 @@ public Builder setAkkaConfiguration(final Map<String, ?> value) {
         @NotEmpty
         private String _httpStatusPath = "/status";
         private Class<? extends SupplementalRoutes> _supplementalHttpRoutesClass;
+        @NotNull
+        private Boolean _logDeadLetters = false;
```

Comment:
tl/dr; That's not the convention in a builder. 

Details: While it's not valid at build time for this to be a null, someone can call `set...` and set it to null. There should be nothing wrong with that. For example, when deserializing from JSON it should not NPE trying to set this to null, but rather fail at construction due to a constraint violation.

Reply:
Why are we not logging dead letters?  Seems like it's a bug if we get dead letters.

Reply:
As mentioned in one of the comments the existing Akka configuration logs only the type of message, recipient and sender. This logs the entire message (via it's `toString`). It was helpful in identifying the messages which were being lost during an in-memory restart. I don't expect to leave it on by default.

Reply:
> While it's not valid at build time for this to be a null, someone can call set... and set it to null. 

That's a little...odd but I suppose I can buy it since the actual runtime model is a primitive.

Reply:
It was not a slam dunk design when we did it 6 years ago, but it's worked reasonably well. Open to suggestions on improving it.